scripts/publish-to-github.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+echo "@dimensiondev:registry=https://npm.pkg.github.com/DimensionDev" > "$HOME/.npmrc"
+echo "//npm.pkg.github.com/:_authToken=${NODE_AUTH_TOKEN}" >> "$HOME/.npmrc"
```

Comment:
Does the token configured to expose to the `master` branch only?

Reply:
https://docs.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token